keyboards/gmmk/gmmk2/p65/ansi/keymaps/vnmm/readme.md

```diff
@@ -0,0 +1,10 @@
+# Vnmm's ANSI GMMKV2 65% Layout
+
+This keymap builds on archrovisual's but with some changes and uses the default key placements
+
+## Features
+
+-   Alphabet keys light up red when caps lock is on
+-   Pressing FN shows keys that have a definition
+-   Via enabled
+-   Quick reset with fn+space
```

Comment:
You dont **need** a readme on your keymap + this info is at userspace too

Reply:
I would prefer to keep it in each keymap, if that is okay. Just in case anyone is looking through and wants to know what it does.